### PR TITLE
Remove purpose (activity) of store during install and in config

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Main activity of store has no purpose for either install and using the store. We have only one set of demo data.
| Type?             | refacto
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| How to test?      | Check that you can install Prestashop normally and shop activity is removed from BO > Shop configuration > General.
| Fixed ticket?     | 
| Related PRs       | https://github.com/PrestaShop/autoupgrade/pull/659
| Sponsor company   | 

### Removed configuration values
- PS_SHOP_ACTIVITY

### CODE REVIEW

**Comment by Hlavtox** [comment]
Ping @MatShir @kpodemski @jf-viguier for validation. :-)

**Review by FabienPapet — APPROVED** [review]

**Comment by jf-viguier** [comment]
ok for me

**Review by MatShir — DISMISSED** [review]
I need to check with the team if the variable is not used. Please wait for the feedback before doing any actions. 🙏

**Comment by PrestaEdit** [comment]
Back to the 8.0, it was not removed due to an external usage by PrestaShop SA, and a long long long discussion was done on this.

https://github.com/PrestaShop/PrestaShop/pull/28853
https://github.com/PrestaShop/PrestaShop/pull/26850

**Comment by Hlavtox** [comment]
PrestaShop SA can implement their own solution via hooks and a custom installer, there is no need to have unused field in the open source distribution.

**Comment by MatShir** [comment]
Yes exactly @PrestaEdit 
I'm checking if the situation has evolved ;)

**Comment by kpodemski** [comment]
any news @MatShir ?

**Comment by MatShir** [comment]
Still digging in the infra :)

**Comment by MatShir** [comment]
The internal team has confirmed that this field is no longer used by our infra 🥳 
For any future data tracking project, it should probably be included in ps_apidistrib (at least through a module).

**Comment by Hlavtox** [comment]
@kpodemski Can you approve? We have a green light!
@MatShir Thanks dude!

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
For QA: auto tests should be more than enough 👍🏻

**Comment by djoelleuch** [comment]
this PR must be tested by a functional  QA, I will remove the `waiting for dev` label

**Comment by Robin-Fischer-PS** [comment]
Indeed, there are functional steps to verify on this one, it should pass through "normal" QA process :)

**Comment by Hlavtox** [comment]
@djoelleuch @Robin-Fischer-PS This PR concerns only the installer and Shop settings > General page, you don't need to test anything else.

**Review by aniszr — APPROVED** [review]
Hello @Hlavtox 

I tested your PR, all improvements are well displayed :

- Removing main activity field in installation menu:heavy_check_mark: 
- Removing main activity from BO > General:heavy_check_mark: 
- Installation process was fluid:heavy_check_mark: 

See attached screenrecord :

https://github.com/PrestaShop/PrestaShop/assets/92626242/75768135-b9f5-47b5-af48-b7799e3fb092

Auto Test : https://github.com/aniszr/ga.tests.ui.pr/actions/runs/5856873529 :heavy_check_mark:  (1 fail related to Nightly)

Thanks!

**Comment by kpodemski** [comment]
thanks @Hlavtox

**Pull request merged**
